src/status_im/transport/filters.cljs

```diff
@@ -0,0 +1,70 @@
+(ns status-im.transport.filters
```

Comment:
Basic 1 sentence docstring explaining what filters even are for (audience: Clojure OSS people who don't know about Whisper).